desktop/package/linux/exchange.haveno.Haveno.yml

```diff
@@ -35,7 +35,7 @@ modules:
       - mkdir -p /app/share/icons/hicolor/128x128/apps/
       - mkdir -p /app/share/applications/
       - mkdir -p /app/share/metainfo/
-      - mv icon.png /app/share/icons/hicolor/128x128/apps/haveno.png
+      - mv exchange.haveno.Haveno.png /app/share/icons/hicolor/128x128/apps/
```

Comment:
To avoid renaming icon.png, which is a dependency for other installers, does it make sense / is it possible to rename icon.png to exchange.haveno.Haveno.png here, and leave icon.png as is?

Reply:
Yep this works, retained the icon.png naming in https://github.com/haveno-dex/haveno/pull/1428/commits/bb57f9ee8552c4f2a8ca63b978010b94e2cc5393